packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1908,6 +2085,7 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
             (bool success, bytes memory returndata) = ovmCALL(
                 _transaction.gasLimit,
                 _transaction.entrypoint,
+                0,
```

Comment:
can also remove this 0

Reply:
on second thought: gonna leave these.  Slightly more efficient and minimizes usage of an effectively deprecated code path